docs/user_manual/components.md

```diff
@@ -369,6 +439,22 @@ However, the reference direction and meaning of `RealValueInput` is different, a
 | `p_specified` | `RealValueInput` | watt (W)                   | specified active power   | &#10024; only for power flow | &#10004; |
 | `q_specified` | `RealValueInput` | volt-ampere-reactive (var) | specified reactive power | &#10024; only for power flow | &#10004; |
 
+##### Electric model
+
+`generic_load_gen` are modelled by using the so-called ZIP load model in power-grid-model, 
+where a load/generator is represented as a composition of constant power (P), constant current (I) and constant impedance (Z).
+
+The injection current I_inj of each ZIP model type can be computed from s = `p_specified` + j`q_specified`as:
+
+For constant impedance (Z) load: I_inj = S*u
```

Comment:
mention that u here is calculated node voltage and it is complex of u and u_angle attribute of node